application/modules/custom_fields/models/Mdl_client_custom.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+if (!defined('BASEPATH')) {
+    exit('No direct script access allowed');
+}
+
+/*
+ * InvoicePlane
+ *
+ * A free and open source web based invoicing system
+ *
+ * @package		InvoicePlane
+ * @author		Kovah (www.kovah.de)
+ * @copyright	Copyright (c) 2012 - 2015 InvoicePlane.com
+ * @license		https://invoiceplane.com/license.txt
+ * @link		  https://invoiceplane.com
+ *
+ */
+
+class Mdl_Client_Custom extends Validator
+{
+    public $table = 'ip_client_custom';
+    public $primary_key = 'ip_client_custom.client_custom_id';
+
+    public function save_custom($client_id, $db_array)
+    {
+        $result = $this->validate($db_array);
+        if ($result === true) {
+            $db_array = $this->_formdata;
+            $client_custom_id = null;
+            $db_array['client_id'] = $client_id;
+            $client_custom = $this->where('client_id', $client_id)->get()->row();
```

Comment:
Can't reproduce. Maybe it was because the field was empty. Can you retry with my latest changes?